Marlin/src/MarlinCore.cpp

```diff
@@ -355,6 +355,9 @@ void enable_all_steppers() {
   ENABLE_AXIS_X();
   ENABLE_AXIS_Y();
   ENABLE_AXIS_Z();
+  #if ENABLED(SOFTWARE_DRIVER_ENABLE)
+    planner.axis_enabled = {true};
```

Comment:
I would rather see this use the following form, so that it is more clear that all three values are being set.
What you have is _likely_ working due to the operators defined on the class, but this looks like a bug if this were normal array initialziation :)

`planner.axis_enabled.set(true, true, true);`

Reply:
Done in last commit.